src/lib/theme.ts

```diff
@@ -6,9 +6,9 @@ export default {
     gray: "#F0F0F0",
   },
   schemes: {
-    dark: { bg: "#14111D", fg: "#ffffff" },
-    light: { bg: "#ffffff", fg: "#14111D" },
-    gray: { bg: "#F0F0F0", fg: "#14111D" },
+    dark: { backgroundColor: "#14111D", color: "#ffffff" },
+    light: { backgroundColor: "#ffffff", fg: "#14111D" },
```

Comment:
Why `backgroundColor` here and not `foregroundColor`?

Reply:
I used `backgroundColor` because that corresponds to the css property. Having `color` as the second item is correct. Going to push up a change setting `fg` to `color` in the other schemes